update idoft

xinying-7 opened the pull request:
fixed a flacky test and updated two project status to developer fixed. Xinying Lyu

darko-marinov commented:
Please make the checker pass

xinying-7 commented:
fixed a new flacky test and updated other changes in status for this week and last week

darko-marinov commented on pr-data.csv:
```diff
@@ -4050,6 +4050,7 @@ https://github.com/fluent/fluent-logger-java,da14ec349bf0904da9865940b57b8756361
 https://github.com/fromage/redpipe,0aff891d6befdf0dcc2bdbfda22262cdf82ac66f,redpipe-engine,net.redpipe.engine.swagger.SwaggerApiTest.checkSwagger,NOD,RepoArchived,,
 https://github.com/fromage/redpipe,0aff891d6befdf0dcc2bdbfda22262cdf82ac66f,redpipe-templating-freemarker,net.redpipe.templating.freemarker.ApiTest.checkMail,NOD,RepoArchived,,
+https://github.com/fromage/redpipe,0aff891d6befdf0dcc2bdbfda22262cdf82ac66f,redpipe-templating-freemarker,net.redpipe.templating.freemarker.ApiTest.checkTemplateNegociationDefault,ID,DeveloperFixed,https://github.com/FroMage/redpipe/commit/2fc605bc2fad4aadbb3b05282f528a05673df2ca#diff-86139da6bd888a5e7589f89d4f81a426bac99cd8aabab363efb5e338c1158f5bR131,
 https://github.com/fromage/redpipe,0aff891d6befdf0dcc2bdbfda22262cdf82ac66f,redpipe-templating-freemarker,net.redpipe.templating.freemarker.ApiTest.checkTemplateNegociationDefault,ID,Opened,https://github.com/fromage/redpipe/pull/56,
```
Where is the evidence (logs) for the claim that this PR fixed this test?

darko-marinov commented on pr-data.csv:
```diff
@@ -4735,6 +4736,7 @@ https://github.com/jenkinsci/jenkins,d2fa87606246e1f1dc1450d7199778343945197e,te
 https://github.com/jenkinsci/jenkins,d2fa87606246e1f1dc1450d7199778343945197e,test,lib.layout.IconTest.testBallColorTd,ID,,,
 https://github.com/jenkinsci/jenkins,d2fa87606246e1f1dc1450d7199778343945197e,test,lib.layout.IconTest.testTasks,ID,DeveloperFixed,,https://github.com/jenkinsci/jenkins/commit/81072df4f7f3c0ecef0d5ab20100a87cb0acce58
 https://github.com/jenkinsci/remoting,abf0455a68ad6c52a57e912bb89d51f883f77542,.,hudson.remoting.ClassFilterTest.capabilityRead,OD,,,https://github.com/TestingResearchIllinois/flaky-test-dataset/issues/82
+https://github.com/jenkinsci/remoting,abf0455a68ad6c52a57e912bb89d51f883f77542,.,hudson.remoting.ClassFilterTest.capabilityRead,OD,DeveloperFixed,,https://github.com/jenkinsci/remoting/commit/26e24490f041ef423cbe1c94b262983790613fbd#diff-ee56285c19352719bba4df32d1496b4c9d9f11c34062a587815d0e1b041b66df
```
Where is the evidence (logs) for the claim that this PR fixed this test?

Please update the previous row (by adding your link into the last column separated with `;` from the existing `..../issues/82`).

darko-marinov commented on pr-data.csv:
```diff
@@ -5515,8 +5517,10 @@ https://github.com/ripe-ncc/whois,79e90f41d7eeee42f2c3ad299bfccc4e8005ce8c,whois
 https://github.com/ripe-ncc/whois,79e90f41d7eeee42f2c3ad299bfccc4e8005ce8c,whois-endtoend,net.ripe.db.whois.spec.domain.AckResponseTest.successes,UD,,,
 https://github.com/robovm/robovm,ef091902377c00dc0fb2db87e8d79c8afb5e9010,compiler,org.robovm.compiler.config.ConfigTest.testWriteConsole,ID,Opened,https://github.com/robovm/robovm/pull/1272,
 https://github.com/robovm/robovm,ef091902377c00dc0fb2db87e8d79c8afb5e9010,compiler,org.robovm.compiler.config.ConfigTest.testWriteConsole,ID,Opened,https://github.com/robovm/robovm/pull/1274,
+https://github.com/robovm/robovm,ef091902377c00dc0fb2db87e8d79c8afb5e9010,compiler,org.robovm.compiler.config.ConfigTest.testWriteConsole,ID,Opened,https://github.com/robovm/robovm/pull/1276,
```
Your PR 1276 seems to be a duplicate of 1274 (and 1272)! Please close your PR 1276 and remove these lines.

darko-marinov commented on pr-data.csv:
```diff
@@ -6627,9 +6631,9 @@ https://github.com/zalando/riptide,dcf4af17abfd312243897e79e0577d1e28537e7d,ript
 https://github.com/zalando/riptide,dcf4af17abfd312243897e79e0577d1e28537e7d,riptide-spring-boot-autoconfigure,org.zalando.riptide.autoconfigure.AsyncClientHttpRequestFactoryTest.shouldAutowireSync,UD,Deleted,,https://github.com/zalando/riptide/commit/3f698fad27f92d7b0df0240d0d5343bb3e293b4b#diff-93984af22d3eb64d6de9635a3c47ef64cac7e7ed745342dae2a823897fd31750
 https://github.com/zalando/riptide,dcf4af17abfd312243897e79e0577d1e28537e7d,riptide-spring-boot-autoconfigure,org.zalando.riptide.autoconfigure.CircuitBreakerTest.shouldAutowireCircuitBreakers,NOD,,,
-https://github.com/zalando/riptide,dcf4af17abfd312243897e79e0577d1e28537e7d,riptide-spring-boot-autoconfigure,org.zalando.riptide.autoconfigure.ClientConfigurationTest.shouldApplyTimeouts,NOD,,,
+https://github.com/zalando/riptide,dcf4af17abfd312243897e79e0577d1e28537e7d,riptide-spring-boot-autoconfigure,org.zalando.riptide.autoconfigure.ClientConfigurationTest.shouldApplyTimeouts,NOD,DeveloperFixed,,https://github.com/zalando/riptide/commit/7374312bfaec5277302d9cd5627e9c477b45e7f6#diff-3a5c9d71b943a3c0b0f272cee25211205cf4ed1f0dbe7301c442f77b02097071R71
```
Where is the evidence (logs) for the claim that this PR fixed this test?

darko-marinov commented:
I'm still waiting for a reply here. We can close this PR for negative "points", or you can provide more info (especially logs). You also need to rebase your changes on the latest `main`.

xinying-7 commented:
I rechecked all these issues in week4, corrected some of my false status changes, and add all the log files required, which are recorded in progress_4.md.   Please refer to PR https://github.com/TestingResearchIllinois/idoft/pull/1464 to see the changes. I directly opened another PR and abandoned this one. （For the latest version that rebased on latest main, see https://github.com/TestingResearchIllinois/idoft/pull/1597)

darko-marinov commented:
Please revise this PR instead of "abandoning" it. Just keep the few lines you want to change but rebase your work on the latest `main` and force push to this PR branch.

xinying-7 commented:
I revised the changes here, since in later weeks I found some of this week's change was problematic, which I deleted, and the rest of the issues were fixed in https://github.com/TestingResearchIllinois/idoft/pull/1464. To avoid duplicate changes with PR 1464, I just removed the lines of PR 1276 here.

darko-marinov commented:
This PR is ready to accept as soon as you squash the two commits into one. If you want some more "points" about this, you should check all cases where the same test is repeated:
```
$ cut -f1,3-6 -d, pr-data.csv | sort | uniq -c | sort -rn | grep -v '^ *1' | grep -v /tinkerpop
```

xinying-7 commented:
I found 8 repeated cases in pr-data.csv:

`      2 https://github.com/robovm/robovm,compiler,org.robovm.compiler.config.ConfigTest.testWriteIOS,ID,Opened
      2 https://github.com/robovm/robovm,compiler,org.robovm.compiler.config.ConfigTest.testWriteConsole,ID,Opened
      2 https://github.com/pholser/junit-quickcheck,core,com.pholser.junit.quickcheck.ExhaustingAGivenSetTest.manyParameters,ID;NIO,Accepted
      2 https://github.com/pholser/junit-quickcheck,core,com.pholser.junit.quickcheck.ExhaustingAGivenSetButIncludingAnotherTest.manyParametersWithBooleanAndEnum,ID;NIO,Accepted
      2 https://github.com/pholser/junit-quickcheck,core,com.pholser.junit.quickcheck.ExhaustingAGivenSetButIncludingAnotherTest.manyParameters,ID;NIO,Accepted
      2 https://github.com/karatelabs/karate,karate-core,com.intuit.karate.core.parser.FeatureParserTest.testEngineForSimpleFeature,ID,DeveloperWontFix
      2 https://github.com/karatelabs/karate,karate-core,com.intuit.karate.core.KarateHttpMockHandlerTest.testMultipleCookies,ID,Rejected
      2 https://github.com/karatelabs/karate,karate-core,com.intuit.karate.core.FeatureResultTest.testJsonConversion,ID,DeveloperWontFix
xinying7@fa24-cs527-094:~/idoft$ cut -f1,3-6 -d, pr-data.csv | sort | uniq -c | sort -rn | grep -v '^ *1' | grep -v /tinkerpop`

I removed all the second one for repeated cases.

darko-marinov commented on pr-data.csv:
```diff
@@ -5038,11 +5038,8 @@ https://github.com/kagkarlsson/db-scheduler,4a8a28e60640dcd03e8d3bf212f483d35b0b
 https://github.com/kal-an/file-reader-app,ba6af3f33000ff1fcde9338922e55a7b1c374f99,.,ru.kalan.filereaderapp.MultiPartParserTest.createBook_ok,ID,Opened,https://github.com/kal-an/file-reader-app/pull/1,
 https://github.com/kanwangzjm/funiture,71ca0fb0658b3d839d9e049ac36429207f05329b,.,com.app.mvc.dao.ShortUrlDaoTest.testSave,NIO,Opened,https://github.com/kanwangzjm/funiture/pull/51,https://github.com/kanwangzjm/funiture/issues/50
 https://github.com/karatelabs/karate,16452ef4d17ed76b079c6f1eba07a4749fa08d02,karate-core,com.intuit.karate.core.FeatureResultTest.testJsonConversion,ID,DeveloperWontFix,,
-https://github.com/karatelabs/karate,2c42468d5f32cb32592ad4a46bcf5ff1a4e60a5e,karate-core,com.intuit.karate.core.FeatureResultTest.testJsonConversion,ID,DeveloperWontFix,,
```
Fine to remove one or the other of these two rows as they just repeat the same info. How did you decide whether to remove `16452ef4d17ed76b079c6f1eba07a4749fa08d02` or `2c42468d5f32cb32592ad4a46bcf5ff1a4e60a5e`? Presumably we want to remove the later commit and keep the earlier one.

darko-marinov commented on pr-data.csv:
```diff
@@ -5038,11 +5038,8 @@ https://github.com/kagkarlsson/db-scheduler,4a8a28e60640dcd03e8d3bf212f483d35b0b
 https://github.com/karatelabs/karate,16452ef4d17ed76b079c6f1eba07a4749fa08d02,karate-core,com.intuit.karate.core.FeatureResultTest.testJsonConversion,ID,DeveloperWontFix,,
-https://github.com/karatelabs/karate,2c42468d5f32cb32592ad4a46bcf5ff1a4e60a5e,karate-core,com.intuit.karate.core.FeatureResultTest.testJsonConversion,ID,DeveloperWontFix,,
 https://github.com/karatelabs/karate,16452ef4d17ed76b079c6f1eba07a4749fa08d02,karate-core,com.intuit.karate.core.KarateHttpMockHandlerTest.testMultipleCookies,ID,Rejected,https://github.com/karatelabs/karate/pull/1856,
-https://github.com/karatelabs/karate,2c42468d5f32cb32592ad4a46bcf5ff1a4e60a5e,karate-core,com.intuit.karate.core.KarateHttpMockHandlerTest.testMultipleCookies,ID,Rejected,https://github.com/karatelabs/karate/pull/2348,
```
Do not remove this row as the overall info differs from the previous (showing two rejected PRs).

darko-marinov commented on pr-data.csv:
```diff
@@ -5038,11 +5038,8 @@ https://github.com/kagkarlsson/db-scheduler,4a8a28e60640dcd03e8d3bf212f483d35b0b
 https://github.com/karatelabs/karate,16452ef4d17ed76b079c6f1eba07a4749fa08d02,karate-core,com.intuit.karate.core.KarateHttpMockHandlerTest.testMultipleCookies,ID,Rejected,https://github.com/karatelabs/karate/pull/1856,
-https://github.com/karatelabs/karate,2c42468d5f32cb32592ad4a46bcf5ff1a4e60a5e,karate-core,com.intuit.karate.core.KarateHttpMockHandlerTest.testMultipleCookies,ID,Rejected,https://github.com/karatelabs/karate/pull/2348,
 https://github.com/karatelabs/karate,16452ef4d17ed76b079c6f1eba07a4749fa08d02,karate-core,com.intuit.karate.core.parser.FeatureParserTest.testEngineForSimpleFeature,ID,DeveloperWontFix,,
-https://github.com/karatelabs/karate,2c42468d5f32cb32592ad4a46bcf5ff1a4e60a5e,karate-core,com.intuit.karate.core.parser.FeatureParserTest.testEngineForSimpleFeature,ID,DeveloperWontFix,,
```
Same comment as above about removing `16452ef4d17ed76b079c6f1eba07a4749fa08d02` or `2c42468d5f32cb32592ad4a46bcf5ff1a4e60a5e` -- which to remove and which to keep?

darko-marinov commented on pr-data.csv:
```diff
@@ -5945,9 +5942,7 @@ https://github.com/pholser/junit-quickcheck,9361b6dae25b193782604d071754eb7fc566
 https://github.com/pholser/junit-quickcheck,9361b6dae25b193782604d071754eb7fc566390a,core,com.pholser.junit.quickcheck.ExhaustingAGivenSetButIncludingAnotherTest.explicitConversion,NIO,Accepted,https://github.com/pholser/junit-quickcheck/pull/357,
 https://github.com/pholser/junit-quickcheck,9361b6dae25b193782604d071754eb7fc566390a,core,com.pholser.junit.quickcheck.ExhaustingAGivenSetButIncludingAnotherTest.favorValueOf,NIO,Accepted,https://github.com/pholser/junit-quickcheck/pull/361,
 https://github.com/pholser/junit-quickcheck,9361b6dae25b193782604d071754eb7fc566390a,core,com.pholser.junit.quickcheck.ExhaustingAGivenSetButIncludingAnotherTest.manyParameters,ID;NIO,Accepted,https://github.com/pholser/junit-quickcheck/pull/284,
-https://github.com/pholser/junit-quickcheck,9361b6dae25b193782604d071754eb7fc566390a,core,com.pholser.junit.quickcheck.ExhaustingAGivenSetButIncludingAnotherTest.manyParameters,ID;NIO,Accepted,https://github.com/pholser/junit-quickcheck/pull/357,
```
Do not delete this row as some info (PR number/link) differs.

darko-marinov commented on pr-data.csv:
```diff
@@ -5945,9 +5942,7 @@ https://github.com/pholser/junit-quickcheck,9361b6dae25b193782604d071754eb7fc566
 https://github.com/pholser/junit-quickcheck,9361b6dae25b193782604d071754eb7fc566390a,core,com.pholser.junit.quickcheck.ExhaustingAGivenSetButIncludingAnotherTest.manyParameters,ID;NIO,Accepted,https://github.com/pholser/junit-quickcheck/pull/284,
-https://github.com/pholser/junit-quickcheck,9361b6dae25b193782604d071754eb7fc566390a,core,com.pholser.junit.quickcheck.ExhaustingAGivenSetButIncludingAnotherTest.manyParameters,ID;NIO,Accepted,https://github.com/pholser/junit-quickcheck/pull/357,
 https://github.com/pholser/junit-quickcheck,9361b6dae25b193782604d071754eb7fc566390a,core,com.pholser.junit.quickcheck.ExhaustingAGivenSetButIncludingAnotherTest.manyParametersWithBooleanAndEnum,ID;NIO,Accepted,https://github.com/pholser/junit-quickcheck/pull/284,
-https://github.com/pholser/junit-quickcheck,9361b6dae25b193782604d071754eb7fc566390a,core,com.pholser.junit.quickcheck.ExhaustingAGivenSetButIncludingAnotherTest.manyParametersWithBooleanAndEnum,ID;NIO,Accepted,https://github.com/pholser/junit-quickcheck/pull/357,
```
Do not delete this row as some info (PR number/link) differs.

darko-marinov commented on pr-data.csv:
```diff
@@ -5972,7 +5967,6 @@ https://github.com/pholser/junit-quickcheck,9361b6dae25b193782604d071754eb7fc566
 https://github.com/pholser/junit-quickcheck,9361b6dae25b193782604d071754eb7fc566390a,core,com.pholser.junit.quickcheck.ExhaustingAGivenSetTest.explicitConversion,NIO,Accepted,https://github.com/pholser/junit-quickcheck/pull/293,
 https://github.com/pholser/junit-quickcheck,9361b6dae25b193782604d071754eb7fc566390a,core,com.pholser.junit.quickcheck.ExhaustingAGivenSetTest.favorValueOf,NIO,Accepted,https://github.com/pholser/junit-quickcheck/pull/293,
 https://github.com/pholser/junit-quickcheck,9361b6dae25b193782604d071754eb7fc566390a,core,com.pholser.junit.quickcheck.ExhaustingAGivenSetTest.manyParameters,ID;NIO,Accepted,https://github.com/pholser/junit-quickcheck/pull/284,
-https://github.com/pholser/junit-quickcheck,9361b6dae25b193782604d071754eb7fc566390a,core,com.pholser.junit.quickcheck.ExhaustingAGivenSetTest.manyParameters,ID;NIO,Accepted,https://github.com/pholser/junit-quickcheck/pull/361,
```
Do not delete this row as some info (PR number/link) differs.

darko-marinov commented on pr-data.csv:
```diff
@@ -6157,10 +6151,7 @@ https://github.com/ripe-ncc/whois,79e90f41d7eeee42f2c3ad299bfccc4e8005ce8c,whois
 https://github.com/ripe-ncc/whois,79e90f41d7eeee42f2c3ad299bfccc4e8005ce8c,whois-api,net.ripe.db.whois.api.rdap.RdapResponseJsonTest.nameserver_serialization_test,ID,Accepted,https://github.com/RIPE-NCC/whois/pull/1097,
 https://github.com/ripe-ncc/whois,79e90f41d7eeee42f2c3ad299bfccc4e8005ce8c,whois-endtoend,net.ripe.db.whois.spec.domain.AckResponseTest.successes,UD,,,
 https://github.com/robovm/robovm,ef091902377c00dc0fb2db87e8d79c8afb5e9010,compiler,org.robovm.compiler.config.ConfigTest.testWriteConsole,ID,Opened,https://github.com/robovm/robovm/pull/1272,
-https://github.com/robovm/robovm,ef091902377c00dc0fb2db87e8d79c8afb5e9010,compiler,org.robovm.compiler.config.ConfigTest.testWriteConsole,ID,Opened,https://github.com/robovm/robovm/pull/1274,
```
Add https://github.com/robovm/robovm/pull/1274 and https://github.com/robovm/robovm/pull/1276 into `Notes` for https://github.com/robovm/robovm/pull/1272

darko-marinov commented on pr-data.csv:
```diff
@@ -6157,10 +6151,7 @@ https://github.com/ripe-ncc/whois,79e90f41d7eeee42f2c3ad299bfccc4e8005ce8c,whois
-https://github.com/robovm/robovm,ef091902377c00dc0fb2db87e8d79c8afb5e9010,compiler,org.robovm.compiler.config.ConfigTest.testWriteConsole,ID,Opened,https://github.com/robovm/robovm/pull/1274,
-https://github.com/robovm/robovm,ef091902377c00dc0fb2db87e8d79c8afb5e9010,compiler,org.robovm.compiler.config.ConfigTest.testWriteConsole,ID,Opened,https://github.com/robovm/robovm/pull/1276,
 https://github.com/robovm/robovm,ef091902377c00dc0fb2db87e8d79c8afb5e9010,compiler,org.robovm.compiler.config.ConfigTest.testWriteIOS,ID,Opened,https://github.com/robovm/robovm/pull/1272,
-https://github.com/robovm/robovm,ef091902377c00dc0fb2db87e8d79c8afb5e9010,compiler,org.robovm.compiler.config.ConfigTest.testWriteIOS,ID,Opened,https://github.com/robovm/robovm/pull/1274,
```
Add https://github.com/robovm/robovm/pull/1274 into `Notes` for https://github.com/robovm/robovm/pull/1272

darko-marinov reviewed (changes requested):
Let's clean these rows properly if you're already looking into all cases of (potential) repetition.

xinying-7 commented:
I added those line back, and for the decision between 16452ef4d17ed76b079c6f1eba07a4749fa08d02  and 2c42468d5f32cb32592ad4a46bcf5ff1a4e60a5e, I indeed decided to keep the earlier one and remove the later one, aligning to the similar case of https://github.com/robovm/robovm/pull/1272, https://github.com/robovm/robovm/pull/1274, https://github.com/robovm/robovm/pull/1276, where PR 1272 is the earliest commit so we keep that one.

darko-marinov commented:
PR 1272 is clearly earlier than PR 1274 or 1276, because we can just compare the numbers. For commit SHAs, such as 16452ef4d17ed76b079c6f1eba07a4749fa08d02 and 2c42468d5f32cb32592ad4a46bcf5ff1a4e60a5e, it's hard to tell which is earlier unless mapping to the date. What were their dates?

xinying-7 commented:
16452ef4d17ed76b079c6f1eba07a4749fa08d02 was on Nov 26, 2021, and 2c42468d5f32cb32592ad4a46bcf5ff1a4e60a5e was on Jan 27, 2023.

darko-marinov commented on pr-data.csv:
```diff
@@ -6156,11 +6154,8 @@ https://github.com/ripe-ncc/whois,79e90f41d7eeee42f2c3ad299bfccc4e8005ce8c,whois
-https://github.com/robovm/robovm,ef091902377c00dc0fb2db87e8d79c8afb5e9010,compiler,org.robovm.compiler.config.ConfigTest.testWriteIOS,ID,Opened,https://github.com/robovm/robovm/pull/1272,
-https://github.com/robovm/robovm,ef091902377c00dc0fb2db87e8d79c8afb5e9010,compiler,org.robovm.compiler.config.ConfigTest.testWriteIOS,ID,Opened,https://github.com/robovm/robovm/pull/1274,
+https://github.com/robovm/robovm,ef091902377c00dc0fb2db87e8d79c8afb5e9010,compiler,org.robovm.compiler.config.ConfigTest.testWriteConsole,ID,Opened,https://github.com/robovm/robovm/pull/1272,https://github.com/robovm/robovm/pull/1274 https://github.com/robovm/robovm/pull/1276
+https://github.com/robovm/robovm,ef091902377c00dc0fb2db87e8d79c8afb5e9010,compiler,org.robovm.compiler.config.ConfigTest.testWriteIOS,ID,Opened,https://github.com/robovm/robovm/pull/1272,https://github.com/robovm/robovm/pull/1274 https://github.com/robovm/robovm/pull/1276
```
Was 1276 for `ConfigTest.testWriteIOS` as well as for `ConfigTest.testWriteConsole`, or was it only for the latter?

xinying-7 commented:
It's for both of them. I fixed two tests in PR 1276. There's 2 commits.

获取 Outlook for Android<https://aka.ms/ghei36>
________________________________
From: darko-marinov ***@***.***>
Sent: Sunday, December 22, 2024 12:47:45 AM
To: TestingResearchIllinois/idoft ***@***.***>
Cc: Lyu, Xinying ***@***.***>; Author ***@***.***>
Subject: Re: [TestingResearchIllinois/idoft] update idoft (PR #1353)


@darko-marinov commented on this pull request.

________________________________

In pr-data.csv<https://urldefense.com/v3/__https://github.com/TestingResearchIllinois/idoft/pull/1353*discussion_r1894793178__;Iw!!DZ3fjg!9yuTwio3nKdGgHbEjb1HdJ4yuAwXbAXvE0t4lpzxmjHJP9lhk-YpmBLj6MLssFjKsgCO2qLWGyXe4ByLSYdYaiDpsVDS$>:

> @@ -6156,11 +6154,8 @@ https://github.com/ripe-ncc/whois,79e90f41d7eeee42f2c3ad299bfccc4e8005ce8c,whois<https://urldefense.com/v3/__https://github.com/ripe-ncc/whois,79e90f41d7eeee42f2c3ad299bfccc4e8005ce8c,whois__;!!DZ3fjg!9yuTwio3nKdGgHbEjb1HdJ4yuAwXbAXvE0t4lpzxmjHJP9lhk-YpmBLj6MLssFjKsgCO2qLWGyXe4ByLSYdYavK4M8dj$>
 https://github.com/ripe-ncc/whois,79e90f41d7eeee42f2c3ad299bfccc4e8005ce8c,whois-api,net.ripe.db.whois.api.rdap.RdapResponseJsonTest.ip_serialization_test,ID,Accepted,https://github.com/RIPE-NCC/whois/pull/1097<https://urldefense.com/v3/__https://github.com/ripe-ncc/whois,79e90f41d7eeee42f2c3ad299bfccc4e8005ce8c,whois-api,net.ripe.db.whois.api.rdap.RdapResponseJsonTest.ip_serialization_test,ID,Accepted,https:/*github.com/RIPE-NCC/whois/pull/1097__;Lw!!DZ3fjg!9yuTwio3nKdGgHbEjb1HdJ4yuAwXbAXvE0t4lpzxmjHJP9lhk-YpmBLj6MLssFjKsgCO2qLWGyXe4ByLSYdYarrKE6lR$>,
 https://github.com/ripe-ncc/whois,79e90f41d7eeee42f2c3ad299bfccc4e8005ce8c,whois-api,net.ripe.db.whois.api.rdap.RdapResponseJsonTest.nameserver_serialization_test,ID,Accepted,https://github.com/RIPE-NCC/whois/pull/1097<https://urldefense.com/v3/__https://github.com/ripe-ncc/whois,79e90f41d7eeee42f2c3ad299bfccc4e8005ce8c,whois-api,net.ripe.db.whois.api.rdap.RdapResponseJsonTest.nameserver_serialization_test,ID,Accepted,https:/*github.com/RIPE-NCC/whois/pull/1097__;Lw!!DZ3fjg!9yuTwio3nKdGgHbEjb1HdJ4yuAwXbAXvE0t4lpzxmjHJP9lhk-YpmBLj6MLssFjKsgCO2qLWGyXe4ByLSYdYaiEKWKHm$>,
 https://github.com/ripe-ncc/whois,79e90f41d7eeee42f2c3ad299bfccc4e8005ce8c,whois-endtoend,net.ripe.db.whois.spec.domain.AckResponseTest.successes,UD<https://urldefense.com/v3/__https://github.com/ripe-ncc/whois,79e90f41d7eeee42f2c3ad299bfccc4e8005ce8c,whois-endtoend,net.ripe.db.whois.spec.domain.AckResponseTest.successes,UD__;!!DZ3fjg!9yuTwio3nKdGgHbEjb1HdJ4yuAwXbAXvE0t4lpzxmjHJP9lhk-YpmBLj6MLssFjKsgCO2qLWGyXe4ByLSYdYaomg0xwB$>,,,
-https://github.com/robovm/robovm,ef091902377c00dc0fb2db87e8d79c8afb5e9010,compiler,org.robovm.compiler.config.ConfigTest.testWriteConsole,ID,Opened,https://github.com/robovm/robovm/pull/1272<https://urldefense.com/v3/__https://github.com/robovm/robovm,ef091902377c00dc0fb2db87e8d79c8afb5e9010,compiler,org.robovm.compiler.config.ConfigTest.testWriteConsole,ID,Opened,https:/*github.com/robovm/robovm/pull/1272__;Lw!!DZ3fjg!9yuTwio3nKdGgHbEjb1HdJ4yuAwXbAXvE0t4lpzxmjHJP9lhk-YpmBLj6MLssFjKsgCO2qLWGyXe4ByLSYdYarKwobmW$>,
-https://github.com/robovm/robovm,ef091902377c00dc0fb2db87e8d79c8afb5e9010,compiler,org.robovm.compiler.config.ConfigTest.testWriteConsole,ID,Opened,https://github.com/robovm/robovm/pull/1274<https://urldefense.com/v3/__https://github.com/robovm/robovm,ef091902377c00dc0fb2db87e8d79c8afb5e9010,compiler,org.robovm.compiler.config.ConfigTest.testWriteConsole,ID,Opened,https:/*github.com/robovm/robovm/pull/1274__;Lw!!DZ3fjg!9yuTwio3nKdGgHbEjb1HdJ4yuAwXbAXvE0t4lpzxmjHJP9lhk-YpmBLj6MLssFjKsgCO2qLWGyXe4ByLSYdYajd2isaa$>,
-https://github.com/robovm/robovm,ef091902377c00dc0fb2db87e8d79c8afb5e9010,compiler,org.robovm.compiler.config.ConfigTest.testWriteConsole,ID,Opened,https://github.com/robovm/robovm/pull/1276<https://urldefense.com/v3/__https://github.com/robovm/robovm,ef091902377c00dc0fb2db87e8d79c8afb5e9010,compiler,org.robovm.compiler.config.ConfigTest.testWriteConsole,ID,Opened,https:/*github.com/robovm/robovm/pull/1276__;Lw!!DZ3fjg!9yuTwio3nKdGgHbEjb1HdJ4yuAwXbAXvE0t4lpzxmjHJP9lhk-YpmBLj6MLssFjKsgCO2qLWGyXe4ByLSYdYaq49D0CK$>,
-https://github.com/robovm/robovm,ef091902377c00dc0fb2db87e8d79c8afb5e9010,compiler,org.robovm.compiler.config.ConfigTest.testWriteIOS,ID,Opened,https://github.com/robovm/robovm/pull/1272<https://urldefense.com/v3/__https://github.com/robovm/robovm,ef091902377c00dc0fb2db87e8d79c8afb5e9010,compiler,org.robovm.compiler.config.ConfigTest.testWriteIOS,ID,Opened,https:/*github.com/robovm/robovm/pull/1272__;Lw!!DZ3fjg!9yuTwio3nKdGgHbEjb1HdJ4yuAwXbAXvE0t4lpzxmjHJP9lhk-YpmBLj6MLssFjKsgCO2qLWGyXe4ByLSYdYamXhHQH4$>,
-https://github.com/robovm/robovm,ef091902377c00dc0fb2db87e8d79c8afb5e9010,compiler,org.robovm.compiler.config.ConfigTest.testWriteIOS,ID,Opened,https://github.com/robovm/robovm/pull/1274<https://urldefense.com/v3/__https://github.com/robovm/robovm,ef091902377c00dc0fb2db87e8d79c8afb5e9010,compiler,org.robovm.compiler.config.ConfigTest.testWriteIOS,ID,Opened,https:/*github.com/robovm/robovm/pull/1274__;Lw!!DZ3fjg!9yuTwio3nKdGgHbEjb1HdJ4yuAwXbAXvE0t4lpzxmjHJP9lhk-YpmBLj6MLssFjKsgCO2qLWGyXe4ByLSYdYagyEL1_o$>,
+https://github.com/robovm/robovm,ef091902377c00dc0fb2db87e8d79c8afb5e9010,compiler,org.robovm.compiler.config.ConfigTest.testWriteConsole,ID,Opened,https://github.com/robovm/robovm/pull/1272,https://github.com/robovm/robovm/pull/1274<https://urldefense.com/v3/__https://github.com/robovm/robovm,ef091902377c00dc0fb2db87e8d79c8afb5e9010,compiler,org.robovm.compiler.config.ConfigTest.testWriteConsole,ID,Opened,https:/*github.com/robovm/robovm/pull/1272,https:/*github.com/robovm/robovm/pull/1274__;Ly8!!DZ3fjg!9yuTwio3nKdGgHbEjb1HdJ4yuAwXbAXvE0t4lpzxmjHJP9lhk-YpmBLj6MLssFjKsgCO2qLWGyXe4ByLSYdYamxInAuV$> https://github.com/robovm/robovm/pull/1276<https://urldefense.com/v3/__https://github.com/robovm/robovm/pull/1276__;!!DZ3fjg!9yuTwio3nKdGgHbEjb1HdJ4yuAwXbAXvE0t4lpzxmjHJP9lhk-YpmBLj6MLssFjKsgCO2qLWGyXe4ByLSYdYarnfeaEq$>
+https://github.com/robovm/robovm,ef091902377c00dc0fb2db87e8d79c8afb5e9010,compiler,org.robovm.compiler.config.ConfigTest.testWriteIOS,ID,Opened,https://github.com/robovm/robovm/pull/1272,https://github.com/robovm/robovm/pull/1274<https://urldefense.com/v3/__https://github.com/robovm/robovm,ef091902377c00dc0fb2db87e8d79c8afb5e9010,compiler,org.robovm.compiler.config.ConfigTest.testWriteIOS,ID,Opened,https:/*github.com/robovm/robovm/pull/1272,https:/*github.com/robovm/robovm/pull/1274__;Ly8!!DZ3fjg!9yuTwio3nKdGgHbEjb1HdJ4yuAwXbAXvE0t4lpzxmjHJP9lhk-YpmBLj6MLssFjKsgCO2qLWGyXe4ByLSYdYat-mdUeM$> https://github.com/robovm/robovm/pull/1276<https://urldefense.com/v3/__https://github.com/robovm/robovm/pull/1276__;!!DZ3fjg!9yuTwio3nKdGgHbEjb1HdJ4yuAwXbAXvE0t4lpzxmjHJP9lhk-YpmBLj6MLssFjKsgCO2qLWGyXe4ByLSYdYarnfeaEq$>


Was 1276 for ConfigTest.testWriteIOS as well as for ConfigTest.testWriteConsole, or was it only for the latter?

―
Reply to this email directly, view it on GitHub<https://urldefense.com/v3/__https://github.com/TestingResearchIllinois/idoft/pull/1353*pullrequestreview-2519553755__;Iw!!DZ3fjg!9yuTwio3nKdGgHbEjb1HdJ4yuAwXbAXvE0t4lpzxmjHJP9lhk-YpmBLj6MLssFjKsgCO2qLWGyXe4ByLSYdYaipkhs3D$>, or unsubscribe<https://urldefense.com/v3/__https://github.com/notifications/unsubscribe-auth/AWDZON6NBZTAZVETAL6EI4T2GZHADAVCNFSM6AAAAABQFD5HC2VHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDKMJZGU2TGNZVGU__;!!DZ3fjg!9yuTwio3nKdGgHbEjb1HdJ4yuAwXbAXvE0t4lpzxmjHJP9lhk-YpmBLj6MLssFjKsgCO2qLWGyXe4ByLSYdYalhd5c3x$>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

darko-marinov commented:
Okay, I've accepted by squashing 2 commits so we need not iterate more.
